scripts/modal.js

```diff
@@ -1,3 +1,133 @@
+let htmlElements = '';
+
+const projects = [
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+  {
+    id: 'projects-wrapper',
+    name: 'Project name goes here',
+    demo: 'https://ksigei.github.io/portfolio2/',
+    source: 'https://github.com/ksigei/portfolio2',
+    langs: {
+      html: 'HTML/CSS',
+      ruby: 'Ruby on Rails',
+      js: 'JavaScript',
+    },
+  },
+];
+
+for (let i = 0; i <= 6; i += 1) {
+  htmlElements
+   += ` 
+   <div class="modal" id="modal2" data-animation="mixInAnimations">
+   <div class="modal-dialog">
+     <header class="modal-header">
+       <button class="close-modal" aria-label="close modal" data-close>
+         ✕  
+       </button>
```

Comment:
- [ ] Kindly style the close button correctly and add it to the desktop version, I noticed even if the user clicks outside the page closes but it would be nice to still have the icon just like the figma design 👍🏿 

 Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![btn](https://user-images.githubusercontent.com/69638013/157829423-38a2fe80-ab81-49ea-8bef-57ff9f74457c.png) |  ![Screenshot_2](https://user-images.githubusercontent.com/69638013/157829375-e996c71a-4846-4e1a-9a5f-0cabc66c53bd.png)